lms/migrations/versions/088813d3e6f8_email_digest_enable_02.py

```diff
@@ -0,0 +1,28 @@
+"""
+Enable email digest (batch 02).
+
+Revision ID: 088813d3e6f8
+Revises: b11713314988
+Create Date: 2023-05-19 10:19:54.691419
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from lms.migrations.versions.b11713314988_email_digest_enable_01 import (
+    enable_email_digest,
+)
+
+
+revision = "088813d3e6f8"
+down_revision = "b11713314988"
+
+
+def upgrade():
+    conn = op.get_bind()
+    enable_email_digest(conn, 2000)
```

Comment:
Being a bit more aggressive now. This is 2000 instances but with very few active teacher (max of 4).